src/Vector.ts

```diff
@@ -325,6 +325,15 @@ export default class Vector {
     return new Vector(entries, dimensions)
   }
 
+  /**
+   * Is it close to zero?
+   * @param eps Euclidean distance tolerance.
+   * @return Checks v ~= 0
+   */
+  isCloseToZero(eps = 1e-6): boolean {
+    return Math.sqrt(this.normSquared()) < eps
+  }
```

Comment:
I see no point in using a variable epsilon. How about also avoiding the sqrt?
```suggestion
  isCloseToZero(): boolean {
    return this.normSquared() < 1e-12
  }
```

This is basically the same as `Complex.isAlmostZero`

Also while we're at it, the naming should be consistent. Can you choose either `isCloseToZero` or `isAlmostZero` and make it consistent across the board?